runner/runners/output.go

```diff
@@ -32,3 +40,61 @@ func (o *nullOutput) Write(p []byte) (int, error) {
 func (o *nullOutput) Close() error {
 	return nil
 }
+
+type localOutputCreator struct {
+	hostname string
+}
+
+// Create a new OutputCreator
+func NewOutputCreator(tmp *temp.TempDir) (runner.OutputCreator, error) {
+	hostname, err := os.Hostname()
+	if err != nil {
+		return nil, err
+	}
+	return &localOutputCreator{hostname: hostname}, nil
```

Comment:
why not use Create method?  This will simplify maintaining this code in the future